From b4680e3bf5dc20a8a87988a537da575096d9fb69 Mon Sep 17 00:00:00 2001
From: Allen Luce <allen.luce@prometheanworld.com>
Date: Thu, 13 Feb 2020 17:29:32 +0000
Subject: [PATCH] Turn AdBlock off by default

This disables the Bromite/NoChromo adblocking engine by default. Ad
blocking causes some sites to not work properly, either due to
ad-blocking detectors or due to broken page handling when ads don't
appear.

Ad blocking can be easily re-enabled via the drop-down hamburger
menu. This should help to ensure that only users that know how to
disable adblocking end up using it in the first place.
---
 .../content_settings/core/browser/content_settings_registry.cc  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/content_settings/core/browser/content_settings_registry.cc b/components/content_settings/core/browser/content_settings_registry.cc
--- a/components/content_settings/core/browser/content_settings_registry.cc
+++ b/components/content_settings/core/browser/content_settings_registry.cc
@@ -342,7 +342,7 @@ void ContentSettingsRegistry::Init() {
            ContentSettingsInfo::EXCEPTIONS_ON_SECURE_AND_INSECURE_ORIGINS);

   Register(ContentSettingsType::ADS, "subresource-filter",
-           CONTENT_SETTING_BLOCK, WebsiteSettingsInfo::UNSYNCABLE,
+           CONTENT_SETTING_ALLOW, WebsiteSettingsInfo::UNSYNCABLE,
            AllowlistedSchemes(),
            ValidSettings(CONTENT_SETTING_ALLOW, CONTENT_SETTING_BLOCK),
            WebsiteSettingsInfo::TOP_ORIGIN_ONLY_SCOPE,
--